docs/getting-started/install.mdx

```diff
@@ -52,9 +52,9 @@ This will add a line like this to your package's pubspec.yaml (and run an implic
 ```yaml
 dependencies:
   widgetbook_annotation: ^{{ versions.annotation }}
-  widgetbook: ^{{ versions.widgetbook }}
 
 dev_dependencies:
+  widgetbook: ^{{ versions.widgetbook }}
```

Comment:
Add `widgetbook` to `dev_dependecies` is not currently possible with the current structure of widgetbook living under the `lib` folder. This will cause the following analyzer warning:

> The imported package 'widgetbook' isn't a dependency of the importing package.
Try adding a dependency for 'widgetbook' in the 'pubspec.yaml' file. [depend_on_referenced_packages](https://dart.dev/lints/depend_on_referenced_packages)

We worked on a new structure that will move `widgetbook` out of the app dependencies, and make it live in its own package. This new structure will be available soon after doing some more research about how breaking it might be.

Reply:
I knew that, however I got an impression from @LucasJosefiak that this is fixed and I tried it was not but I still though maybe something is baking that I’m not aware that is why I sent the PR.

let’s then either close this or draft it until we have this change in place 

thanks

Reply:
I will close this for now, until we internally discuss the new structure more.
Thanks for the effort anyway, @mhadaily!